src/Components/Pages/FindLocationPage/FindLocationPage.jsx

```diff
@@ -1,7 +1,70 @@
+import { useEffect, useState } from 'react';
+import { Outlet, useNavigate, useParams } from 'react-router';
+import { getLocationById, getCurrentWeather } from '../../../Api/weatherApi';
+import LocationSearch from '../../LocationSearch/LocationSearch';
+import CurrentLocation from '../../CurrentLocation/CurrentLocation';
+import { capitalize } from '../../../helpers/formatHelper';
+import SafeWeatherImage from '../../SafeImage/SafeWeatherImage';
+
 function FindLocation() {
+  const navigate = useNavigate();
+  const { locationId } = useParams();
+
+  const [selectedLocation, setSelectedLocation] = useState();
+  const [currentWeather, setCurrentWeather] = useState();
+  const [error, setError] = useState();
+
+  useEffect(async () => {
+    if (locationId) {
+      await invokeAction(async () => {
```

Comment:
Please read doc "How to use async functions in useEffect"
https://devtrium.com/posts/async-functions-useeffect